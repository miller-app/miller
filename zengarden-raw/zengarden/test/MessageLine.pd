#N canvas 677 54 548 463 10;
#X msg 319 179 0 1000;
#X msg 411 145 1 1000;
#X obj 229 319 line 0 100;
#X obj 64 12 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 101 17 loadbang;
#X obj 229 350 print;
#X msg 473 144 0;
#X msg 235 177 1;
#X obj 78 50 t b b b b b b;
#X obj 147 147 delay 2000;
#X msg 147 177 0 1000;
#X msg 60 176 stop;
#X obj 60 146 delay 2500;
#X obj 235 147 delay 2000;
#X obj 319 147 delay 950;
#X connect 0 0 2 0;
#X connect 1 0 2 0;
#X connect 2 0 5 0;
#X connect 3 0 8 0;
#X connect 4 0 8 0;
#X connect 6 0 2 0;
#X connect 7 0 2 0;
#X connect 8 0 12 0;
#X connect 8 1 9 0;
#X connect 8 2 13 0;
#X connect 8 3 14 0;
#X connect 8 4 1 0;
#X connect 8 5 6 0;
#X connect 9 0 10 0;
#X connect 10 0 2 0;
#X connect 11 0 2 0;
#X connect 12 0 11 0;
#X connect 13 0 7 0;
#X connect 14 0 0 0;
