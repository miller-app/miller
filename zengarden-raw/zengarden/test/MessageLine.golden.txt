[@ 0.000ms] print: 0
[@ 0.000ms] print: 0
[@ 100.000ms] print: 0.1
[@ 200.000ms] print: 0.2
[@ 300.000ms] print: 0.3
[@ 400.000ms] print: 0.4
[@ 500.000ms] print: 0.5
[@ 600.000ms] print: 0.6
[@ 700.000ms] print: 0.7
[@ 800.000ms] print: 0.8
[@ 900.000ms] print: 0.9
[@ 950.000ms] print: 0.95
[@ 1050.000ms] print: 0.855
[@ 1150.000ms] print: 0.76
[@ 1250.000ms] print: 0.665
[@ 1350.000ms] print: 0.57
[@ 1450.000ms] print: 0.475
[@ 1550.000ms] print: 0.38
[@ 1650.000ms] print: 0.285
[@ 1750.000ms] print: 0.19
[@ 1850.000ms] print: 0.095
[@ 1950.000ms] print: 0
[@ 2000.000ms] print: 1
[@ 2000.000ms] print: 1
[@ 2100.000ms] print: 0.9
[@ 2200.000ms] print: 0.8
[@ 2300.000ms] print: 0.7
[@ 2400.000ms] print: 0.6
